doc/modules/model_evaluation.rst

```diff
@@ -317,6 +317,106 @@ parameter:
+  - *Which scoring function is a good one for my task?*
+
+In a nutshell, if the scoring function is given, e.g. in a kaggle competition
+or in a business context, use that one.
+If you are free to choose, it starts by considering the ultimate goal and application
+of the prediction. It is useful to distinguish two steps:
+
+* Predicting
+* Decision making
+
+**Prediction:**
+Usually, the response variable :math:`Y` is a random variable, in the sense that there
+is *no deterministic* function :math:`Y = g(X)`.
+Instead, there is a probability distribution :math:`F` of :math:`Y`.
+One can aim to predict the whole distribution, known as *probabilistic prediction*,
+or---more the focus of scikit-learn---issue a *point prediction* (or point forecast)
+by choosing a property or functional of that distribution :math:`F`.
+Typical examples are the mean (expected value), the median or a quantile of the
+response variable :math:`Y` (conditionally on :math:`X`).
+
+Once that is settled, use a **strictly consistent** scoring function for that
+target functional, see [Gneiting2009]_.
+For classification **stricly proper scoring rules**, see [Gneiting2007]_, conincide
+with strictly consistent scoring functions.
+One could say that consistent scoring functions act as *truth serum* in that
+they guarantee "that truth telling [. . .] is an optimal strategy in
+expectation" [Gneiting2014]_.
+
+Note that for regressors, the prediction ist done with :term:`predict` while for
+classifiers it is usually :term:`predict_proba`.
+
+**Decision Making:**
+The most common decisions are done on binary classification tasks, where the result of
+:term:`predict_proba` is turned into a single outcome, e.g., from the predicted
+probability of rain a decision is made on how to act (whether to take mitigating
+measures like an umbrella or not).
+For classifiers, this is what :term:`predict` returns.
+See also :ref:`TunedThresholdClassifierCV`.
+There are many scoring functions which measure different aspects of such a
+decision, most of them are covered with or derived from the :func:`confusion_matrix`.
+
+**List of strictly consistent scoring functions:**
+Here, we list some of the most well known statistical functionals and corresponding
+stritcly consistent scoring functions. Note that the list is not complete and that
+there are more of them.
+For further criteria on how to select a specific one, see [Fissler2022]_.
+
+==================  ========================  ================  ==============================
+functional          scoring or loss function  response y        prediction
+==================  ========================  ================  ==============================
+**Classification**
+mean                Brier score               multi-class       ``predict_proba``
+mean                log loss                  multi-class       ``predict_proba``
+mode                zero-one loss :sup:`1`    multi-class       ``predict``, categorical
```

Comment:
I'm wondering if should have the "categorical", "all reals", etc. (I would refer those to property but it seems that you had another definition for this term) in a separate column and refer to "prediction" as the scikit-learn prediction method.

Reply:
What is your suggestion exactly?

Reply:
I would think at something like:

```
==========  ========================  ==========  ================= ========
functional  scoring or loss function  response y  prediction method property
==========  ========================  ==========  ================= ========
**Classification**
mean                Brier score               multi-class       ``predict_proba``       real in [0, 1]
mean                log loss                  multi-class       ``predict_proba``       real in [0, 1]
mode                zero-one loss :sup:`1`    multi-class       ``predict``       categorical
```

Note that I'm really not sure about the column names that I proposed.